spec/sidekiq/evss/disability_compensation_form/submit_form0781_spec.rb

```diff
@@ -861,4 +861,52 @@
       end
     end
   end
+  describe '#get_docs' do
+    let(:submission_id) { 1 }
+    let(:uuid) { 'some-uuid' }
+    let(:submission) { create(:form526_submission, id: submission_id) }
```

Comment:
Using build `{ build(:form526_submission, id: submission_id) }` here instead would optimize test performance unless you need this record to exist in the database. `build` is significantly faster compared to create, especially in large test suites.